Shared/Samples/Augment reality to navigate route/AugmentRealityToNavigateRouteView.swift

```diff
@@ -16,6 +16,8 @@ import ArcGIS
 import CoreLocation
 import SwiftUI
 
+@available(macCatalyst, unavailable)
+@MainActor
```

Comment:
```suggestion
```

Is this needed? I believe conforming to `View` already puts this on the main actor.

Reply:
Yes, this was accidentally included when I resolved the rebase merge conflict in #539